tmt/package_managers/bootc.py

```diff
@@ -0,0 +1,262 @@
+import json
+import re
+import tempfile
+import uuid
+from typing import Any, Optional, cast
+
+import tmt.utils
+from tmt.package_managers import (
+    FileSystemPath,
+    Installable,
+    Options,
+    PackageManager,
+    PackageManagerEngine,
+    dnf,
+    provides_package_manager,
+)
+from tmt.utils import Command, CommandOutput, GeneralError, Path, RunError, ShellScript
+
+
+class BootcEngine(PackageManagerEngine):
+    def __init__(self, *args: Any, **kwargs: Any) -> None:
+        """Initialize bootc engine for package management"""
+        super().__init__(*args, **kwargs)
+
+        self.aux_engine = dnf.DnfEngine(*args, **kwargs)
+
+        self.containerfile_directives = self._get_base_containerfile_directives()
```

Comment:
This should be a call to `self.initialize_containerfile_directives()` - if you have a method that initializes an internal state, you should use it and nothing else.